pkg/sync/rpc/server.go

```diff
@@ -1000,8 +1002,12 @@ func (s *SyncAgentServer) purgeSnapshots() (err error) {
 		s.PurgeStatus.Progress = int(float32(removed) / float32(markedRemoved) * 100)
 		s.PurgeStatus.Unlock()
 	}
+
+	// Purge is done, clear total and processed
```

Comment:
Why do we need to do this?

Reply:
do you mean the comment or the code to clear total and processed?
We need to clear the processed, currently the reason we have this issue is because after the first purge, the processed becomes to 1, and then we hit the over 100% issue.

Also I will add more explanation on the commit message.

Reply:
Oh, so it's prepared for the next purge? That makes sense, but it should also done by the purge initialization part as well.